src/client/routes.jsx

```diff
@@ -1,22 +1,22 @@
 import React from 'react';
 import { Route } from 'react-router';
 
-import AdditionalInformationSection from './components/personal-information/AdditionalInformationSection';
+// import AdditionalInformationSection from './components/veteran-information/AdditionalInformationSection';
```

Comment:
Did you mean to comment this out? Don't we want this to now be under `insurance-information`?